package.json

```diff
@@ -49,7 +49,7 @@
     "express": "^4.17.2",
     "hydrogen-view-sdk": "npm:@mlm/hydrogen-view-sdk@^0.20.0-scratch",
     "json5": "^2.2.1",
-    "linkedom": "^0.14.17",
+    "linkedom": "^0.14.18",
```

Comment:
Doesn't seem like this will move forward given `linkedom` is going to drop support for `<select>`/`<option>` again, https://github.com/WebReflection/linkedom/issues/171#issuecomment-1287307383